ts/Extensions/ScrollablePlotArea.ts

```diff
@@ -254,10 +254,22 @@ Chart.prototype.setUpScrolling = function (): void {
         attribs.overflowY = 'auto';
     }
 
+    // Insert a container with position relative
+    // that scrolling and fixed container renders to (#10555)
+    const relContainer = createElement(
+        'div',
+        void 0,
```

Comment:
Would consider adding a class name `highcharts-scrolling-parent` or similar just in case someone needs access to this container. Maybe also store the element on the chart object